spec/cable/handler_spec.cr

```diff
@@ -143,7 +143,7 @@ describe Cable::Handler do
       request = HTTP::Request.new("GET", "/unknown_route?test_token=1", headers)
 
       io_with_context = create_ws_request_and_return_io_and_context(handler, request)[0]
-      io_with_context.to_s.should eq("")
```

Comment:
Not sure if this is meant to be nothing, but when the server hits a route that doesn't exist, we get back the 404.